src/components/OpenList/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { useState } from 'react';
+
+const OpenList = () => {
```

Comment:
I should have removed this shit. I will do it in the next commit @albertofernandezmrf